src/mmw/js/src/modeling/models.js

```diff
@@ -751,6 +758,7 @@ var ScenariosCollection = Backbone.Collection.extend({
             aoi_census: aoi_census
         });
 
+        scenario.set('waitToRefresh', true);
```

Comment:
Once `waitToRefresh` is set to `true`, `if (this.get('waitToRefresh') === true)` will _always_ resolve to `true`, even if the `parse` is called by the framework, and all resets will be silent as per https://github.com/WikiWatershed/model-my-watershed/pull/957/files#diff-9abeff034ba5a68b4814ee90721888ebR575. If we want to be conservative, then at some point (when we know we are done) we should set it to `false`.

Or, instead of not firing the `reset` event, we might be able to change the spinners to listen to a different signal, which can be manually triggered, just like `reset` is in https://github.com/WikiWatershed/model-my-watershed/pull/957/files#diff-9abeff034ba5a68b4814ee90721888ebR660.